coach.yml

```diff
@@ -1,6 +1,6 @@
 project_type: ruby_gem
-ruby_version: 2.4.2
+ruby_version: 2.6.1
 resource_class: small
 databases:
-  postgresql:
-    version: 9.4
+  - type: postgresql
+    version: 9.6
```

Comment:
Coach does NOT support `9.4` anymore for Postgres, also the syntax for databases changed a bit